### PR TITLE
Switch logging to ginkgo writer and misc changes

### PR DESCRIPTION
**Pull request by harsh-px**
1) Move logging to ginkgo writer
2) Add default CheckIfPathExists for aws node driver
3) Switch to failFast runs

### CODE REVIEW

File: util/util.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+func nowStamp() string {
+	return time.Now().Format(time.StampMilli)
+}
+
+func log(level string, format string, args ...interface{}) {
+	fmt.Fprintf(ginkgo.GinkgoWriter, nowStamp()+": "+level+": "+format+"\n", args...)
+}
+
+// Logf logs info log to ginkgo writer
+func Logf(format string, args ...interface{}) {
```
Comment (piyush-nimbalkar) [inline util/util.go]: Instead of Logf, we should have individual methods like Infof, Warnf, Errorf

File: drivers/node/aws/aws.go [hunk 2]
```diff
@@ -180,6 +180,11 @@ func (a *aws) ShutdownNode(n node.Node, options node.ShutdownNodeOpts) error {
 	return nil
 }
 
+// TODO add AWS implementation for this
+func (a *aws) CheckIfPathExists(path string, n node.Node, options node.ConnectionOpts) (bool, error) {
```
Comment (piyush-nimbalkar) [inline drivers/node/aws/aws.go]: I might change this in the current review. It should not matter a lot other that the signature change

File: pkg/k8sops/k8sops.go [hunk 3]
```diff
@@ -640,18 +641,23 @@ func (k *k8sOps) ValidateTerminatedStatefulSet(statefulset *apps_api.StatefulSet
 			if matched, _ := regexp.MatchString(".+ not found", err.Error()); matched {
 				return "", nil
 			}
+
+			logrus.Errorf("[debug] error while getting stateful set: %v", err)
```
Comment (piyush-nimbalkar) [inline pkg/k8sops/k8sops.go]: why not use util.logf here?

File: scripts/do-gofmt.sh [hunk 4]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+find_files() {
```
Comment (piyush-nimbalkar) [inline scripts/do-gofmt.sh]: let's remove the check-gofmt.sh if we are going to use this one

File: util/util.go (hunk 1 above)
Comment (harsh-px) [reply util/util.go]: Updated in new incremental

File: scripts/do-gofmt.sh (hunk 4 above)
Comment (harsh-px) [reply scripts/do-gofmt.sh]: Done

File: pkg/k8sops/k8sops.go (hunk 3 above)
Comment (harsh-px) [reply pkg/k8sops/k8sops.go]: Added util.Errorf in new incremental

File: Makefile [hunk 5]
```diff
@@ -19,7 +19,7 @@ GOFMT := gofmt
 
 .DEFAULT_GOAL=all
 
-all: vet lint build fmt
+all: clean vet lint build fmt
```
Comment (piyush-nimbalkar) [inline Makefile]: you may not want to do a clean every time you build, right?
doing make clean && make all could be a better way around this one. What do you think?

File: util/util.go (hunk 1 above)
Comment (piyush-nimbalkar) [reply util/util.go]: I was thinking of renaming Logf to Infof like the other methods or other logging libraries

File: Makefile (hunk 5 above)
Comment (harsh-px) [reply Makefile]: I was going for a default clean to happen every time. But `make clean && make all` works too. Removed it.

File: util/util.go (hunk 1 above)
Comment (harsh-px) [reply util/util.go]: Yeah `Infof` makes more sense. Updated.

**Pull request merged**